libr/core/cmd_anal.c

```diff
@@ -8286,25 +8323,248 @@ R_API void r_core_agraph_print (RCore *core, int use_utf, const char *input) {
+		} else {
+			encbody = r_base64_encode_dyn ("", 0);
+		}
+
+		r_cons_printf ("agn \"%s\" base64:%s\n", print_node->title, encbody);
+		free (encbody);
+	}
+	r_list_foreach (graph->nodes, it, node) {
+		print_node = node->data;
+		r_list_foreach (node->out_nodes, edge_it, target) {
+			RGraphNodeInfo *to = target->data;
+			r_cons_printf ("age \"%s\" \"%s\"\n", print_node->title, to->title);
+		}
+	}
+}
+
+static char *print_graph_dot(RCore *core, RGraph /*RGraphNodeInfo*/ *graph) {
+	RList *nodes = graph->nodes;
+	RListIter *it, *itt;
+	RGraphNode *node = NULL, *target = NULL;
+	RStrBuf buf;
+	const char *font = r_config_get (core->config, "graph.font");
+	r_strbuf_init (&buf);
+	r_strbuf_appendf (&buf,
+		"digraph code {\nrankdir=LR;\noutputorder=edgesfirst\ngraph [bgcolor=azure];\n"
+		"edge [arrowhead=normal, color=\"#3030c0\" style=bold weight=2];\n"
+		"node [fillcolor=white, style=filled shape=box "
+		"fontname=\"%s\" fontsize=\"8\"];\n",
+		font);
+
+	r_list_foreach (nodes, it, node) {
+		RGraphNodeInfo *print_node = (RGraphNodeInfo *)node->data;
+		const char *body = print_node->body;
+		if (!body || !*body) {
+			r_strbuf_appendf (&buf, "%d [URL=\"%s\", color=\"lightgray\", label=\"%s\"]\n",
+				node->idx, print_node->title, print_node->title);
+		} else {
+			r_strbuf_appendf (&buf, "%d [URL=\"%s\", color=\"lightgray\", label=\"%s\\n%s\"]\n",
+				node->idx, print_node->title, print_node->title, body);
+		}
+		r_list_foreach (node->out_nodes, itt, target) {
+			r_strbuf_appendf (&buf, "%d -> %d\n", node->idx, target->idx);
+		}
+	}
+	r_strbuf_append (&buf, "}\n");
+	return r_strbuf_drain_nofree (&buf);
+}
+
+static void r_core_graph_print(RCore *core, RGraph /*<RGraphNodeInfo>*/ *graph, int use_utf, bool use_offset, const char *input) {
+	RAGraph *agraph = NULL;
+	RListIter *it;
+	RListIter *edge_it;
+	RGraphNode *graphNode, *target;
+	RGraphNodeInfo *print_node;
+	if (use_utf != -1) {
+		r_config_set_i (core->config, "scr.utf8", use_utf);
+	}
+	switch (*input) {
+	case 0:
+	case 't':
+	case 'k':
+	case 'v':
+	case 'i': {
+		agraph = create_agraph_from_graph (graph);
+		switch (*input) {
+		case 0:
+			agraph->can->linemode = r_config_get_i (core->config, "graph.linemode");
+			agraph->can->color = r_config_get_i (core->config, "scr.color");
+			r_agraph_set_title (agraph,
+				r_config_get (core->config, "graph.title"));
+			r_agraph_print (agraph);
+			break;
+		case 't': { // "ag_t" - tiny graph
+			agraph->is_tiny = true;
+			int e = r_config_get_i (core->config, "graph.edges");
+			r_config_set_i (core->config, "graph.edges", 0);
+			r_core_visual_graph (core, agraph, NULL, false);
+			r_config_set_i (core->config, "graph.edges", e);
+			break;
+		}
+		case 'k': // "ag_k"
+		{
+			Sdb *db = r_agraph_get_sdb (agraph);
+			char *o = sdb_querys (db, "null", 0, "*");
+			r_cons_print (o);
+			free (o);
+			break;
+		}
+		case 'v': // "ag_v"
+		case 'i': // "ag_i" - open current core->graph in interactive mode
```

Comment:
Is this missing from the help? I see `v Interactive ascii art` but I can't find `i`.